lib/main.dart

```diff
@@ -176,7 +218,85 @@ class StepThreeScreen extends StatelessWidget {
             child: OutlineButton(
               child: Text('Exit'),
               onPressed: () => Coordinator.of(context).next(
-                after: DummyFlowEvent.stepThree(140),
+                after: PersonalInfoFlowStep.weight(140),
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class SomethingSubflow extends StatelessWidget {
```

Comment:
added in a pointless subflow to show how these coordinators stack on top of each other super cleanly